libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.kt

```diff
@@ -1156,44 +1096,8 @@ open class OAuthWebviewHelper : KeyChainAliasCallback {
             SalesforceSDKManager.getInstance().shouldLogoutWhenTokenRevoked()
         )
 
-        // Create account name (shown in Settings -> Accounts & sync)
-        val accountName = buildAccountName(
-            accountOptions?.username,
-            accountOptions?.instanceUrl
-        )
-
         // New account
-        val extras = clientManager.createNewAccount(
-            accountName,
-            accountOptions?.username,
-            accountOptions?.refreshToken,
-            accountOptions?.authToken,
-            accountOptions?.instanceUrl,
-            loginOptions.loginUrl,
-            accountOptions?.identityUrl,
-            oAuthClientId,
-            accountOptions?.orgId,
-            accountOptions?.userId,
-            accountOptions?.communityId,
-            accountOptions?.communityUrl,
-            accountOptions?.firstName,
-            accountOptions?.lastName,
-            accountOptions?.displayName,
-            accountOptions?.email,
-            accountOptions?.photoUrl,
-            accountOptions?.thumbnailUrl,
-            accountOptions?.additionalOauthValues,
-            accountOptions?.lightningDomain,
-            accountOptions?.lightningSid,
-            accountOptions?.vfDomain,
-            accountOptions?.vfSid,
-            accountOptions?.contentDomain,
-            accountOptions?.contentSid,
-            accountOptions?.csrfToken,
-            accountOptions?.nativeLogin,
-            accountOptions?.language,
-            accountOptions?.locale
-        )
+        val extras = clientManager.createNewAccount(account)
```

Comment:
Old code creates new account from accountOptions instead of the account passed to the method.
Not sure why since the only caller to that method is the one creating accountOptions from the account.
Why do we have accountOptions??